solver/src/pending_transactions.rs

```diff
@@ -32,18 +39,35 @@ pub enum Fee {
     },
 }
 
-// Get the pending transactions from a node.
-//
-// This works better than rust-web3's `block_with_txs` because some nodes set the `miner` field to
-// `null`. This does not follow the ethrpc specification and fails deserialization in rust-web3.
+// Get pending transactions from a mempool.
 pub async fn pending_transactions(transport: &DynTransport) -> Result<Vec<Transaction>> {
-    let params = vec!["pending".into(), true.into()];
-    let response = transport
-        .execute("eth_getBlockByNumber", params)
+    match transport
+        .execute("txpool_content", Default::default())
         .await
-        .context("transport failed")?;
-    let block: Block = serde_json::from_value(response).context("deserialize failed")?;
-    Ok(block.transactions)
+        .context("transport failed")
```

Comment:
ubernit: This `context` shouldn't be needed as the error gets ignored and dropped :point_down: anyway.

Reply:
Fixed